mitodl/open-discussions
# infinite drawer

#### Pre-Flight checklist

<img width="1653" alt="Screen Shot 2022-08-25 at 10 19 34 PM" src="https://user-images.githubusercontent.com/1934992/186908889-0a9ea158-a702-4904-8297-85b62650a927.png">
<img width="1654" alt="Screen Shot 2022-08-26 at 8 57 38 AM" src="https://user-images.githubusercontent.com/1934992/186908824-1c5a8b2f-d8b8-4c7c-a7bf-08296bcf147a.png">
<img width="378" alt="Screen Shot 2022-08-25 at 10 20 21 PM" src="https://user-images.githubusercontent.com/1934992/186908877-515ec3f0-bb70-4cb0-acd5-04373ae01c67.png">

#### What are the relevant tickets?
[(Required)
](https://github.com/mitodl/open-discussions/issues/3697)

#### What's this PR do?
This PR adds a course drawer to the infinite corridor

#### How should this be manually tested?
Go to the infinite search page or field page in infinite corridor. You should be able to click on the cards to open the infinite drawer. The drawer should look like the design https://projects.invisionapp.com/d/main#/console/22082155/468407666/preview in desktop
